docs/hugo/content/contributing/developer-setup.md

```diff
@@ -10,11 +10,11 @@ description: "How to set up your developer environment for Azure Service Operato
 
 We support a number of different approaches to ASO development.
 
-- Dev Container with VS Code on Linux
-- Dev Container with VS Code on Windows
-- Docker on Linux
-- CLI on Linux
-- CLI on MacOS
+- [Dev Container with VS Code on Linux](#dev-container-with-vs-code-on-linux)
+- [Dev Container with VS Code on Windows](#dev-container-with-vs-code-on-windows)
+- [Docker on Linux](#docker-on-linux)
+- [CLI on Linux](#cli-on-linux)
+- [CLI on MacOS](#cli-on-macos)
```

Comment:
With this change, I reckon statement on line 84(You'll need to manually install the tools as listed by `.devcontainer/install-dependencies.sh`) proves to be false. Would be great to update that section as well.

Reply:
I've updated macOS instructions. You can still use either `.devcontainer` approach or to install tools manually, executing `.devcontainer/install-dependencies.sh` script which downloads all necessary tools to the localhost.
I tested both scenarios on my apple silicon-based mac.